api/graphql/tests/test_reservation_units/test_reservation_units_state_filter.py

```diff
@@ -222,3 +222,33 @@ def test_filtering_by_hidden(self):
         assert_that(self.content_is_empty(content)).is_false()
         assert_that(content.get("errors")).is_none()
         self.assertMatchSnapshot(content)
+
+    def test_filtering_by_scheduled_publishing_when_begin_after_end(self):

```

Comment:
Just interested: in which case having begin time being after end time makes sense? 🤔

Reply:
When it's scheduled for publishing 🥁 So this case is when the reservation unit it is scheduled for closing at some time and to be republished after that.